units/en/unit3/hands-on.mdx

```diff
@@ -0,0 +1,320 @@
+# Hands-on [[hands-on]]
+
+
+
+      <CourseFloatingBanner classNames="absolute z-10 right-0 top-0"
+      notebooks={[
+        {label: "Google Colab", value: "https://colab.research.google.com/github/huggingface/deep-rl-class/blob/master/notebooks/unit3/unit3.ipynb"}
+        ]}
+        askForHelpUrl="http://hf.co/join/discord" />
+
+
+Now that you've studied the theory behind Deep Q-Learning, **you’re ready to train your Deep Q-Learning agent to play Atari Games**. We'll start with Space Invaders, but you'll be able to use any Atari game you want 🔥
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit4/atari-envs.gif" alt="Environments"/>
+
+
+We're using the [RL-Baselines-3 Zoo integration](https://github.com/DLR-RM/rl-baselines3-zoo), a vanilla version of Deep Q-Learning with no extensions such as Double-DQN, Dueling-DQN, and Prioritized Experience Replay.
+
+
+To validate this hands-on for the certification process, you need to push your trained model to the Hub and **get a result of >= 500**.
+
+To find your result, go to the leaderboard and find your model, **the result = mean_reward - std of reward**
+
+For more information about the certification process, check this section 👉 https://huggingface.co/deep-rl-course/en/unit0/introduction#certification-process
+
+**To start the hands-on click on Open In Colab button** 👇 :
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/huggingface/deep-rl-class/blob/master/notebooks/unit3/unit3.ipynb)
+
+
+# Unit 3: Deep Q-Learning with Atari Games 👾 using RL Baselines3 Zoo
```

Comment:
This notebook is very high level compared to the previous one, so maybe it's good to set up the expectations since the beginning in a more explicit way that we're not going to implement a neural network here but use a common RL library used out there. Maybe at the end, apart from the link to the implementation, we could share some link to a tutorial for implementing Deep Q Learning from scratch?